tracing/src/main/java/com/palantir/tracing/Tracers.java

```diff
@@ -382,7 +406,7 @@ public static Runnable wrapWithAlternateTraceId(String traceId, Runnable delegat
     public static Runnable wrapWithAlternateTraceId(String traceId, String operation, Runnable delegate) {
         return wrapWithAlternateTraceId(traceId, operation, Observability.UNDECIDED, delegate);
     }
-
+/*
```

Comment:
hmm

Reply:
@dansanduleac we do have a copy of all endpoints one for Runnable one for Callable. I don't see why wrapWithAlternateTraceId shouldn't :-P

Reply:
i mean, there's no whitespace and the comment starts on column 0

Reply:
🤦‍♂